src/__tests__/state-management.spec.ts

```diff
@@ -341,4 +370,46 @@ describe('state management', () => {
       })
     })
   })
+
+  describe('sendTransaction', () => {
+    it('should poll transaction until resolved', async () => {
+      const responseSubject = new ReplaySubject<any>()
+
+      walletClient
+        .sendTransaction({
+          transactionManifest: '',
+          version: 1,
+        })
+        .map((response) => {
```

Comment:
I think we want `_unsafeUnwrap()` here?

Reply:
Not sure how it would help me at this point. I don't want to block the execution since I'm sending the wallet response further down.